api/helpers/db.js

```diff
@@ -303,7 +290,7 @@ function media(mediaList) {
   return simpleArray((mediaList || []).map(aMedium).filter(x => !!x)); // remove nulls
 }
 
-function sourcedMedia(mediaList) {
+function sourcedMedia(mediaList, key) {
```

Comment:
doesn't look like this function uses the key param.

Reply:
I use it sometimes for debugging. Can remove it.